tests/unit/cli/statistics_importer.py

```diff
@@ -0,0 +1,173 @@
+import responses
+import unittest
+from pathlib import Path
+
+from rfhub2.cli.statistics_importer import StatisticsImporter
+from rfhub2.cli.api_client import Client
+
+FIXTURE_PATH = Path.cwd() / "tests" / "fixtures" / "statistics"
+EXPECTED_GET_EXECUTION_PATHS = {
+    FIXTURE_PATH / "output.xml",
+    FIXTURE_PATH / "subdir" / "output.xml",
+}
+
+STATISTICS_1 = {
+    "collection": "BuiltIn",
+    "keyword": "Log",
+    "execution_time": "2019-12-25 11:38:08.868000",
+    "times_used": 2,
+    "total_elapsed": 2,
+    "min_elapsed": 1,
+    "max_elapsed": 1,
+}
+STATISTICS_2 = {
```

Comment:
As above, you could reuse first statistics dict fields in the next dicts to make it more concise

Reply:
Done